CAM Platform/api-providers.md

```diff
@@ -1041,2323 +841,561 @@ ElasticBox-Release: 4.0
-                        },
-                        {
-                            "description": "2 vCPUs, 13 GB RAM",
-                            "name": "n1-highmem-2"
-                        },
-                        {
-                            "description": "32 vCPUs, 208 GB RAM",
-                            "name": "n1-highmem-32"
-                        },
-                        {
-                            "description": "4 vCPUs, 26 GB RAM",
-                            "name": "n1-highmem-4"
-                        },
-                        {
-                            "description": "8 vCPUs, 52 GB RAM",
-                            "name": "n1-highmem-8"
-                        },
-                        {
-                            "description": "1 vCPU, 3.75 GB RAM",
-                            "name": "n1-standard-1"
-                        },
-                        {
-                            "description": "16 vCPUs, 60 GB RAM",
-                            "name": "n1-standard-16"
-                        },
-                        {
-                            "description": "2 vCPUs, 7.5 GB RAM",
-                            "name": "n1-standard-2"
-                        },
-                        {
-                            "description": "32 vCPUs, 120 GB RAM",
-                            "name": "n1-standard-32"
-                        },
-                        {
-                            "description": "4 vCPUs, 15 GB RAM",
-                            "name": "n1-standard-4"
-                        },
-                        {
-                            "description": "8 vCPUs, 30 GB RAM",
-                            "name": "n1-standard-8"
-                        }
-                    ],
-                    "name": "us-central1-b"
-                },
-                {
-                    "machineTypes": [
-                        {
-                            "description": "1 vCPU (shared physical core) and 0.6 GB RAM",
-                            "name": "f1-micro"
-                        },
-                        {
-                            "description": "1 vCPU (shared physical core) and 1.7 GB RAM",
-                            "name": "g1-small"
-                        },
-                        {
-                            "description": "16 vCPUs, 14.4 GB RAM",
-                            "name": "n1-highcpu-16"
-                        },
-                        {
-                            "description": "2 vCPUs, 1.8 GB RAM",
-                            "name": "n1-highcpu-2"
-                        },
-                        {
-                            "description": "32 vCPUs, 28.8 GB RAM",
-                            "name": "n1-highcpu-32"
-                        },
-                        {
-                            "description": "4 vCPUs, 3.6 GB RAM",
-                            "name": "n1-highcpu-4"
-     
```

Comment:
Please make sure that the IDs and names you enter are not valid in prod or beta.